core/src/main/java/org/jdbi/v3/core/statement/StatementContext.java

```diff
@@ -86,18 +88,28 @@ public class StatementContext implements Closeable {
     private Instant executionMoment;
     private Instant completionMoment;
     private Instant exceptionMoment;
+    private long mappedRows;
+    private String traceId;
 
-    static StatementContext create(ConfigRegistry config, ExtensionMethod extensionMethod) {
-        final StatementContext context = new StatementContext(config, extensionMethod);
+    static StatementContext create(final ConfigRegistry config, final ExtensionMethod extensionMethod, final Class<?> jdbiStatementType) {
+        final StatementContext context = new StatementContext(config, extensionMethod, jdbiStatementType);
         context.notifyContextCreated();
         return context;
     }
 
-    private StatementContext(ConfigRegistry config, ExtensionMethod extensionMethod) {
+    private StatementContext(final ConfigRegistry config, final ExtensionMethod extensionMethod, final Class<?> jdbiStatementType) {
+        this.jdbiStatementType = jdbiStatementType;
```

Comment:
can we keep those lines in order? The third parameter should be the third line. Can this value be null?